connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectStreamingQueryCache.scala

```diff
@@ -182,12 +179,10 @@ private[connect] class SparkConnectStreamingQueryCache(
             log.info(s"Removing references for $id in session ${v.sessionId} after expiry period")
             queryCache.remove(k)
 
-          case Some(_) => // Inactive query waiting for expiration. Keep the session alive.
-            sessionsToKeepAlive.add((v.userId, v.sessionId))
+          case Some(_) => // Inactive query waiting for expiration. Do nothing.
+            log.info(s"Waiting for the expiration for $id in session ${v.sessionId}")
```

Comment:
This should be logInfo, also I think all other logs in this file should be logInfo.